src/payment/payment.ts

```diff
@@ -80,6 +82,10 @@ export interface PaypalInstrument {
     };
 }
 
+export interface BayclardcardInstrument {
+    browserInfo: BrowserInfoRequest;
```

Comment:
just realised that the idea behind `formattedPayload` is that the payload is already formatted for bigpay https://github.com/bigcommerce/bigpay-client-js/pull/82, to avoid mapping the content in bigpay. So what you had before was actually aligned with what we started 🤦‍♂